development-tools/index.rst

```diff
@@ -1,6 +1,6 @@
-==============
-Advanced Tools
-==============
+=================
+Development Tools
+=================
```

Comment:
Hm, would that work, given that there was no anchor before?

Reply:
See https://github.com/python/devguide/pull/1149#discussion_r1283525322

I can also add anchors, just in case :)

Reply:
All titles automatically create an anchor: https://devguide.python.org/advanced-tools/#advanced-tools

Technically by changing the title you are breaking the automatically-generated anchor and links to that specific section.  This can be fixed by adding an explicit `.. _advanced-tools:` before the title.

In this specific case I wouldn't bother though.  It's unlikely that someone is linking to that specific header rather than the page itself, and even if they are and we break their anchor, the end result will be the same since the header is already at the top of the page.

Reply:
> All titles automatically create an anchor: https://devguide.python.org/advanced-tools/#advanced-tools

I was not aware; TIL!

> In this specific case I wouldn't bother though. It's unlikely that someone is linking to that specific header rather than the page itself, and even if they are and we break their anchor, the end result will be the same since the header is already at the top of the page.

Good point.